core/src/Config.purs

```diff
@@ -142,6 +143,21 @@ backendConfigCodec = CAR.object "BackendConfig"
   , args: CAR.optional (CA.array CA.string)
   }
 
+type PackageBuildOptionsInput =
+  { censor_project_warnings :: Maybe CensorBuildWarnings
+  , censor_project_codes :: Maybe (NonEmptySet.NonEmptySet String)
+  , filter_project_codes :: Maybe (NonEmptySet.NonEmptySet String)
+  , strict :: Maybe Boolean
```

Comment:
Doesn't `pedantic-packages` belong here as well?

Reply:
`pedantic-packages` is not a PSA arg. I do want to move it into the package config, but in a separate PR as I imagine this will affect the order of outputs (e.g. "Build succeeded" messages).

Reply:
Not a PSA arg, but the block I'm commenting on is the `build:` section of the package, which should contain that one as well. But indeed, let's have that change in a separate PR

Reply:
Tracking this here: https://github.com/purescript/spago/issues/1049